tests/integration/models/api-key.spec.ts

```diff
@@ -124,6 +124,31 @@ describe('API Key model', function () {
 		});
 	});
 
+	describe('balena.models.apiKey.getProvisioningApiKeysByApplication', function () {
+		givenLoggedInUser(before);
+
+		before(function () {
+			return balena.models.application
+				.generateApiKey('SampleOrg/sampleApp')
+				.then((apiKey) => {
+					expect(apiKey).to.be.a('string').that.has.length(32);
+
+					this.apiKey = apiKey;
+				});
+		});
+
+		describe('balena.models.apiKey.getProvisioningApiKeysByApplication()', function () {
+			return balena.models.apiKey
+				.getProvisioningApiKeysByApplication('SampleOrg/sampleApp')
+				.then(function (apiKeys) {
+					expect(apiKeys).to.be.an('array');
+					expect(apiKeys).to.have.lengthOf(1);
+					expect(apiKeys[0].name).to.equal('SampleOrg/sampleApp');
+					expect(apiKeys[0].id).to.equal(this.apiKey);
+				});
```

Comment:
I think you missed this
```suggestion
				.then((apiKeys) => {
					expect(apiKeys).to.be.an('array');
					expect(apiKeys).to.have.lengthOf(1);
					expect(apiKeys[0].name).to.equal(null); // but it might be `''`, plz double check.
					expect(apiKeys[0].id).to.equal(this.apiKey.id);
				});
```